### ISSUE TITLE
Showing a single article by pushing an article view controller doesn't render the article on the first try

### ISSUE THREAD

**Issue by jeffremer** [zendesk/zendesk_sdk_ios#441]
If you try to view an article in a freshly installed app for the first time it doesn't appear. This is very confusing and a poor user experience, especially as a first time user of an app looking for help content. 

I first filed [this issue](https://github.com/zendesk/zendesk_sdk_ios/issues/390) four months ago. I posted that the issue persists over a week ago and haven't received a reply so I'm opening a new issue.

After months of back and forth @KillianSmith0 claimed it was fixed. It did seem to be fixed in version 2.1.0 of the SDK. Either it was never actually fixed or it is back in 2.1.1 since I can easily reproduce it both in our production app which is used by millions of users and in [a sample case](https://github.com/jeffremer/ZendeskWarning) (updated with 2.1.1).

- SDK version: 2.1.1
- Xcode version: 10.0 (10A255)
- iOS or Simulator: Both
- iOS Version: iOS 12
- Swift version: 4.2

Steps to Reproduce:

1. If you have the app installed, delete it and reinstall
2. Create a article view controller for an article you haven't seen before
2. Push the article view controller via -[UINavigationController pushViewController:animated:]

Expected: the user should see an article
Actual result: no article, just a blank view controller

![blank](http://g.recordit.co/82OqpmtcUr.gif)

Some sample code, essentially straight from your docs (details redacted):

```
#import "ViewController.h"

@import ZendeskCoreSDK;
@import ZendeskSDK;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [ZDKZendesk initializeWithAppId:@"appID" clientId:@"secret" zendeskUrl:@"https://support.strava.com"];
    [ZDKSupport initializeWithZendesk:[ZDKZendesk instance]];
    id<ZDKObjCIdentity> userIdentity = [[ZDKObjCAnonymous alloc] initWithName:nil email:nil];
    [[ZDKZendesk instance] setIdentity:userIdentity];
}

- (IBAction)showArticle:(id)sender
{
    UIViewController *articleController = [ZDKHelpCenterUi buildHelpCenterArticleWithArticleId:@"216919427"
                                                                                    andConfigs:@[]];
    [self.navigationController pushViewController:articleController animated:YES];
}

@end
```

**Comment by KillianSmith0** [comment]
Hi Jeff, sorry about the delay on getting back to you. We've been busy on a different project, and as a result a few issues have slipped through the cracks. Part of this is that we generally don't see updates on closed issues. 

We did think it was fixed as of 2.1.0, but obviously it must have regressed. We have a tech debt item in our backlog to rewrite the ArticleView completely, which should give us an opportunity to address this (and other) issues more completely. I'll bring this up with our product manager to see if we can bump that up in the list, but in the meantime we'll look into another hotfix to hopefully get it sorted sooner.

**Comment by KillianSmith0** [comment]
Hi @jeffremer,

We've released v2.3 of the SDK, it includes a a fix for the article view not rendering first time round.. Updating should resolve the issue. I'll keep this issue open for now, if there's any other feedback, otherwise will close later in the week. 

Regards,
Killian
